index.html

```diff
@@ -1,227 +1,293 @@
-        <div class="grid g2" id="inverse">
-            <div class="box2">
-                <h1 class="multi">
-                    Multi-Post Stories
-                </h1>
-                <div class="facebook">
-                <p class="p1">FACEBOOK</p><p class="p2">Full Stack Dev</p><p class="p3">2015</p>
-                </div>
-                <h2 class="face">
-                    Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
-                </h2>
-                <div class="buttons">
-                    <ul>
-                        <li><button>html</button></li>
-                        <li><button>css</button></li>
-                        <li><button>javascript</button></li>
-                    </ul>
-                </div>
-                <div class="project">
-                    <button><a href="">See project</a></button>
-                </div>
-            </div>
-            <div class="box1">
-                <img class="image1" src="/images/imagen2.png">
-            </div>
-        </div>
-        <div class="grid g6"></div>
-        <div class="grid g7"></div>
-        <div class="grid g2">
-            <div class="box1">
-                <img class="image1" src="/images/imagen3.png">
-            </div>
-            <div class="box2">
-                <h1>
-                    Facebook 360
-                </h1>
-                <div class="facebook">
-                <p class="p1">FACEBOOK</p><p class="p2">Full Stack Dev</p><p class="p3">2015</p>
-                </div>
-                <h2 class="face">
-                    Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
-                </h2>
-                <div class="buttons">
-                    <ul>
-                        <li><button>html</button></li>
-                        <li><button>css</button></li>
-                        <li><button>javascript</button></li>
-                    </ul>
-                </div>
-                <div class="project">
-                    <button><a href="">See project</a></button>
-                </div>
-            </div>
-        </div>
-        <div class="grid g9"></div>
-        <div class="grid g10"></div>
-        <div class="grid g2" id="inverse">
-            <div class="box2">
-                <h1>
-                    Uber Navigation
-                </h1>
-                <div class="uber">
-                <p class="p1">UBER</p><p class="p2">Lead Developer</p><p class="p3">2018</p>
+      <div class="grid g1"></div>
+      <div class="grid g2">
+        <div class="box1">
+          <img class="image1" src="/images/imagen1.png" />
+        </div>
+        <div class="box2">
+          <h1>Tonic</h1>
+          <div class="canopy">
+            <p class="p1">CANOPY</p>
+            <p class="p2">Back End Dev</p>
```

Comment:
- [ ] It is required to stick to the figma design as close as possible. The `Back End Dev` is on one line on the figma design but split to the next line on your design. Kindly look into it and fix.
- [ ] Also, try to correct this wherever it happened on your project cards.

### Your Design
![your-tonic](https://user-images.githubusercontent.com/26597432/228669994-e1162d04-a2f7-497f-8a34-f3e35d031871.png)

### Figma Design
![temp-tonic](https://user-images.githubusercontent.com/26597432/228670013-ce9b581a-e246-4e1b-a0d5-9e4bf35cd263.png)